MaxText/input_pipeline/_tfds_data_processing_c4_mlperf.py

```diff
@@ -274,7 +278,7 @@ def preprocess_eval_dataset(
     num_examples: Optional[int] = None,
 ) -> tf.data.Dataset:
   """Preprocess the evaluation dataset."""
-  eval_ds = sequence_packing.pack_dataset(eval_ds, max_target_length)
+  eval_ds = sequence_packing.pack_dataset(eval_ds, max_target_length, pad_id=0)
```

Comment:
default value is -1, why we need 0 here? can we use same  pad_id for both train and eval?

Reply:
Made eval the same as train now. Please review again.